aim/web/ui/src/utils/app/getParamsSuggestions.ts

```diff
@@ -0,0 +1,24 @@
+import { IProjectsModelState } from 'types/services/models/projects/projectsModel';
+
+/**
+ * @returns memoized params suggestions from projectsData
+ */
+
+export function getParamsSuggestions(data: IProjectsModelState) {
+  let list: string[] = [];
+  if (data?.params) {
+    Object.keys(data?.params).forEach((option: any) => {
+      if (option) {
+        list.push(`run.${option}`);
+        if (data.params) {
```

Comment:
I think here can be used an operator and one if statement with two conditions. data?.params[option] can be changed to data.params[option] if data hasn't param property or param prop is falsely value this scenario will not be executed.